gatsby-site/src/components/Contact/contact.css

```diff
@@ -0,0 +1,85 @@
+:root {
+    --colarkfontcolor: #0111FC;
```

Comment:
I just noticed this, but is there a way to have these root defs defined in a shared global.css file that is included in all the pages? If all of these values are repeated in each css file, then you don't get as much benefits for using css variables.

Reply:
I'm taking care of it in my global-css pr, coming soon =)